common/model-views.cpp

```diff
@@ -4791,7 +4791,19 @@ namespace rs2
                 if (dev.supports(RS2_CAMERA_INFO_CAMERA_LOCKED))
                     is_locked = std::string(dev.get_info(RS2_CAMERA_INFO_CAMERA_LOCKED)) == "YES" ? true : false;
 
-                if (dev.is<rs2::updatable>() && !is_locked)
+
+                // L515 do not support update unsigned image currently
```

Comment:
Let's do this inside the `if` at line 4806?

Reply:
OK, 
note that this code only runs when the device menu is opened.

Reply:
Done